react/client/src/components/AffirmationComponent.tsx

```diff
@@ -40,8 +39,10 @@ const AffirmationComponent = ({
   description,
   onChangeAffirmation,
 }: ComponentProps) => {
+  const classes = useStyles(isActive);
```

Comment:
This works for now because there is only one prop that needs to be passed, but wondering how this would be handled when there are multiple props needed for dynamic styling.
Will research more but if you know of a more standard way to do this please let me know

Reply:
Looks like you can pass in a prop object instead of just the single variable https://material-ui.com/styles/api/#returns-3